config/environments/development.rb

```diff
@@ -10,7 +10,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false
```

Comment:
Hi @equivalentideas  My bad..The purpose of that line was to test the behavior locally it should be changed I guess.